src/marks/tip.js

```diff
@@ -155,6 +155,7 @@ export class Tip extends Mark {
       .call(applyIndirectStyles, this, dimensions, context)
       .call(applyIndirectTextStyles, this)
       .call(applyTransform, this, {x: X && x, y: Y && y})
+      .style("visibility", "hidden") // avoid flickering
```

Comment:
We should prefer attr here instead of style. (We only use style when attr is not supported.)

```suggestion
      .attr("visibility", "hidden") // hide until postrender
```

Reply:
My thinking was that this would be orthogonal if we wanted to support the visibility attribute. But yeah, we can see this when we (or… if we ever) support it.